projects/kit/components/floating/floating.less

```diff
@@ -0,0 +1,95 @@
+@import '@taiga-ui/core/styles/taiga-ui-local.less';
+
+/**
+ * @name Floating
+ * @selector [tuiFloating]
+ *
+ * @description
+ * Container for creating sticky footers
+ *
+ * @example
+ * <footer tuiFloating>
+ *   <button
+        type="button"
+        tuiButton
+    >
+        Close
+    </button>
+ * </footer>
+ *
+ * @see-also
+ * Block, SheetDialog
+ */
+[tuiFloating] {
+    position: sticky;
+    bottom: max(
+        calc(100 * var(--tui-viewport-vh) - var(--tui-viewport-height) - var(--tui-viewport-y)),
+        calc(env(safe-area-inset-bottom))
+    );
+    z-index: 1;
+    padding: 0 1rem 1rem;
```

Comment:
We should get rid of built-in horizontal padding and instead extend ::before to stick out. Just gotta make sure it does not introduce horizontal scroll to the parent.